packages/box/box.1.2.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Render boxes in the terminal"
+description: """
+Render boxes in the terminal, can configure the padding, margin and the border of the box.
+"""
+maintainer: ["davesnx"]
+authors: ["davesnx"]
+license: "MIT"
+homepage: "https://github.com/davesnx/ocaml-box"
+bug-reports: "https://github.com/davesnx/ocaml-box/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.7" & >= "2.0"}
+  "odoc" {with-doc}
+  "reason" {build}
+  "fmt" {dev}
+]
+dev-repo: "git+https://github.com/davesnx/ocaml-box.git"
+# We need to avoid "@runtest", since it depends on rely
```

Comment:
You could vendor `rely` if that's the only dependency that's not in opam

Reply:
It's a test dependency, does it make sense to vendor it in this case?

Reply:
how so? A vendored package with dune won't be built unless it is needed (only when building the tests in this case)

Reply:
I'm confused then, why did you vendor rely in ocaml-box then? Is it preferred that any consumable of a package can build the tests as well?

Reply:
I’m unsure what you mean by "consumable of a package" but for opam-repository it is better when packages have tests, especially for libraries as this also detects if all the dependencies linked correctly.

My PR got merged (thanks! ^^) so it’ll be done for next release i suppose.